src/templates/header.html

```diff
@@ -1,52 +1,84 @@
-                        aria-expanded="false" aria-controls="gov-banner">
-                        <span class="usa-banner-button-text">Here's how you know</span>
-                    </button>
-                </div>
-            </header>
-            <div class="usa-banner-content usa-accordion-content" id="gov-banner">
-                <div class="grid-row grid-gap-lg">
-                    <div class="usa-banner-guidance-gov tablet:grid-col-6">
-                        <img class="usa-banner-icon usa-media_block-img" src="{{ uswds.path }}/img/icon-dot-gov.svg" alt="Dot gov">
-                        <div class="usa-media_block-body">
-                            <p>
-                                <strong>The .gov means it’s official.</strong>
-                                <br>
-                                Federal government websites often end in .gov or .mil. Before sharing sensitive information, make sure you’re on a federal government site.
-                            </p>
-                        </div>
-                    </div>
-                    <div class="usa-banner-guidance-ssl tablet:grid-col-6">
-                        <img class="usa-banner-icon usa-media_block-img" src="{{ uswds.path }}/img/icon-https.svg" alt="Https">
-                        <div class="usa-media_block-body">
-                            <p>
-                                <strong>The site is secure.</strong>
-                                <br>
-                                The <strong>https://</strong> ensures that you are connecting to the official website and that any information you provide is encrypted and transmitted securely.
-                            </p>
-                        </div>
-                    </div>
-                </div>
+<a class="usa-skipnav" href="#main-content">Skip to main content</a>
+<div class="usa-banner usgs-banner">
+    <header class="usa-banner-header">
+        <div class="usa-banner-inner">
+            <div class="grid-col-auto">
+                <img class="usa-banner-header-flag" src="{{ uswds.path }}img/favicons/favicon-57.png" alt="U.S. flag">
+            </div>
+            <div class="grid-col-fill tablet:grid-col-auto">
+                <p class="usa-banner-header-text">An official website of the United States government</p>
             </div>
         </div>
-    </div>
-
-    <!-- This adds the USGS banner -->
-    <div class="usgs-banner-container content-container">
-        <div class="usgs-logo">
-            <a href="https://www.usgs.gov"><img alt="USGS Home" src="{{ url_for('img/USGS_logo.svg') }}" /></a>
-        </div>
-    </div>
-</div>
+    </header>
+<div class="usa-overlay"></div>
+<header class="usa-header usa-header-extended usgs-header" role="banner">
+   <div class="usa-navbar">
+      <div class="usa-logo">
+        <img alt="USGS Home" src="{{ url_for('img/USGS_logo.svg') }}" />
+        <!--em class="usa-logo-text">
```

Comment:
Should this be removed? I think the USGS vis id standards don't allow anything other than the search box (if needed) in the header with the logo?